src/main/java/hudson/plugins/translation/ResourceBundleFactoryImpl.java

```diff
@@ -30,20 +30,31 @@ protected Properties wrapUp(String locale, Properties props) {
             protected Properties get(String key) {
                 int mc = reloadModCount;
```

Comment:
this access needs to be synchronized if you have changed the field type from volatile